### PR TITLE
Update options page body copy

### PR DESCRIPTION
**Pull request by phacks**
**Before**

CleanShot 2023-06-02 at 14 22 52@2x

**After**

CleanShot 2023-06-02 at 14 22 39@2x

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+  "description": "The Orbit extension brings all your community data right at your fingertips",
   "repository": "git@github.com:orbit-love/orbit-browser-extension.git",
-  "author": "Nicolas Goutay <nicolas.goutay@gmail.com>",
+  "contributors": [
+    {
+      "name": "Nicolas Goutay",
+      "email": "nicolas.goutay@gmail.com"
+    },
+    {
+      "name": "delete-44",
```
Comment (delete-44) [inline package.json:12]: Omg I love that guy

File: src/options/options.html [hunk 2]
```diff
@@ -10,6 +10,28 @@
...
+          alt=""
+          src="../icons/orbit-logo-color-1x.png"
+          srcset="
+            ../icons/orbit-logo-color-2x.png 2x,
+            ../icons/orbit-logo-color-3x.png 3x
+          "
+          style="height: 30px; max-width: inherit"
+        />
+      </a>
+      <iframe
```
Comment (delete-44) [inline src/options/options.html:26]: Excellent idea ^^

**Review by delete-44 — APPROVED** [review]
Is a nice improvement and no blockers for it going in, but thought there would be a screenshot?

File: package.json (hunk 1 above)
Comment (phacks) [reply package.json:12]: he’s great i should introduce you someday you’ll have a blast

**Comment by phacks** [comment]
Ah yes—for the screenshot I’m waiting for the materials Nick and Jenneviere are working on. It’ll no doubt be better than what I can come up with 😄 We can add it on Monday

**Pull request merged**